variables.tf

```
@@ -200,3 +200,9 @@ variable "transport_protocol" {
     error_message = "Invalid protocol type must be one of: udp, tcp."
   }
 }
+
+variable "permissions_boundary" {
+  type        = string
+  default     = ""
```

Can this default to null instead? 

```suggestion
  default     = null
```